### PR TITLE
Improve comment in symex-goto

### PR DESCRIPTION
**Pull request by owen-mc-diffblue**
- [ ] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- [ ] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: src/goto-symex/symex_goto.cpp [hunk 1]
```diff
@@ -216,7 +216,7 @@ void goto_symext::symex_goto(statet &state)
     return;
   }
 
-  // put into state-queue
+  // put a copy of the current state into the state-queue
   framet::goto_state_listt &goto_state_list =
     state.top().goto_state_map[new_state_pc];
```
Comment (tautschnig) [inline src/goto-symex/symex_goto.cpp]: Maybe a slightly weird ask, but would you mind removing the blank line (line 222) here? The comment really only makes sense when also looking at the `goto_state_list.emplace_back(state);` instruction, but that's currently "the next paragraph."

**Review by tautschnig — APPROVED** [review]

File: src/goto-symex/symex_goto.cpp [hunk 2]
```diff
@@ -216,7 +216,7 @@ void goto_symext::symex_goto(statet &state)
     return;
   }
 
-  // put into state-queue
+  // put a copy of the current state into the state-queue
```
Comment (smowton) [inline src/goto-symex/symex_goto.cpp]: Add `, to be used by merge_gotos when we visit new_state_pc`

**Review by smowton — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: d0153a8a2ea8646a1557b5f214639da89a663e1a).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/102538992

File: src/goto-symex/symex_goto.cpp (hunk 1 above)
Comment (owen-mc-diffblue) [reply src/goto-symex/symex_goto.cpp]: You've reminded me that I also wanted to give `state_pc` and `new_state_pc` better names. Hmmm.

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: 811f91d800ecc319b0da73f5c0c47146cdce2e33).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/102630377

**Pull request merged**